underfs/obs/src/main/java/alluxio/underfs/obs/OBSOutputStream.java

```diff
@@ -42,7 +44,7 @@
  * local disk and copied as a complete file when the {@link #close()} method is called.
  */
 @NotThreadSafe
-public final class OBSOutputStream extends OutputStream {
+public final class OBSOutputStream extends OutputStream implements UnderFileSystemOutputStream {
```

Comment:
Can you try to move the contentHash function into OutputStream and have default implementation, the logics seem similar

Reply:
I couldn't figure out a good way to do this, since different classes extend different types out output streams I think I have to use an interface. Unless you can think of a better way?